.markdownlint.yaml

```diff
@@ -34,7 +34,7 @@ MD006: true
 # MD007/ul-indent - Unordered list indentation
 MD007:
   # Spaces for indent
-  indent: 2
+  indent: 4
```

Comment:
@macpijan we can't use 2 as it is in standard markdown since mkdocs requires 4 space indent.

Reply:
roger that